tgui/packages/tgui/interfaces/PowerTransmissionLaser.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * @file
+ * @copyright 2020
+ * @author Sovexe (https://github.com/Sovexe)
+ * @license ISC
+ */
+
+import { useBackend } from '../backend';
+import { Box, Button, ColorBox, Divider, Knob, LabeledList, NoticeBox, ProgressBar, Section } from '../components';
+import { formatMoney, formatPower, formatSiUnit } from '../format';
+import { Window } from '../layouts';
+
+export const PowerTransmissionLaser = (props, context) => {
+  const { data } = useBackend(context);
+  const {
+    lifetimeEarnings,
+    name,
```

Comment:
Fine I think in this case, just noting that sensible defaults for this sort of thing are probably good.
```suggestion
    name = 'Power Transmission Laser',
```